docker/all-in-one/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG postgres_version=15.1.0.83
+ARG postgres_version=15.1.0.136
```

Comment:
```suggestion
ARG postgres_version=15.1.0.148
```

Maybe we could bump into latest? Anyway, not a blocker since everything runs again.